src/client/components/Header.jsx

```diff
@@ -1,12 +1,14 @@
+import { Link } from "react-router-dom";
+
 const Header = () => {
     return (
         <header className="homeHeader">
             <div className="headerTitle"></div>
             <div className="headerNav">
             <ul>
-                <li>Profile</li>
-                <li>Login</li>
-                <li>Ratings</li>
+                <li><Link to="/home">Home</Link></li>
+                <li><Link to="/profile">Profile</Link></li>
+                <li><Link to="/ratings">Ratings</Link></li>
```

Comment:
Nice links

Reply:
looks like frontend routing is coming to life...